scripts/test_version.kts

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env kscript
```

Comment:
I think there's little need for this script because you can test on a Bazel version matrix on Bazel CI already: https://github.com/bazelbuild/continuous-integration/tree/master/buildkite#using-a-specific-version-of-bazel

See an example for rules_jvm_external here https://github.com/bazelbuild/rules_jvm_external/blob/fae429a0c3a5c0da4d4380eeaf347f5331083a5f/.bazelci/presubmit.yml#L17-L48

Reply:
So, the script can and should go, regardless (was going to do that in a followup) because I moved it to cgruber/scripts.  But the tool is useful outside of CI, because once you've moved to a particular version, you don't want to keep failing builds in CI, so the script was meant to be a tool used for documentation, that doesn't require a full CI matrix.  But yeah - I'll kill it.